src/test/system/st_rados_create_pool.cc

```diff
@@ -130,3 +130,18 @@ std::string get_temp_pool_name(const char* prefix)
   ceph_assert((unsigned int)ret < sizeof(poolname));
   return poolname;
 }
+
+std::string get_temp_suffix(const char* _suffix)
+{
+  ceph_assert(_suffix);
+  char hostname[80];
+  int ret = 0;
+  ret = gethostname(hostname, sizeof(hostname));
+  ceph_assert(!ret);
+  char suffix[256];
+  ret = snprintf(suffix, sizeof(suffix),
+                 "%s-%s-%d", _suffix, hostname, getpid());
```

Comment:
Shouldn't we ensure the termination zero?

Reply:
we have ceph_assert(ret > 0);